### PR TITLE
[WIP]new formulae: claws-mail

### PR DESCRIPTION
**Pull request by yurikoles**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----
It can't find `<gdk/gdkx.h>`
<details><summary> brew install claws-mail error</summary>

```
$ brew install claws-mail 
Updating Homebrew...
==> Using the sandbox
==> Downloading http://www.claws-mail.org/download.php?file=releases/claws-mail-3.15.0.tar.xz
Already downloaded: /Users/yurikoles/Library/Caches/Homebrew/claws-mail-3.15.0.tar.xz
==> ./configure --disable-silent-rules --prefix=/usr/local/Cellar/claws-mail/3.15.0
==> make install
Last 15 lines from /Users/yurikoles/Library/Logs/Homebrew/claws-mail/02.make:
#include <gdk/gdkx.h>
         ^
1 error generated.
make[5]: *** [x11/libcmnpgtkhotkey_la-tomboykeybinder.lo] Error 1
make[5]: *** Waiting for unfinished jobs....
x11/eggaccelerators.c:24:10: fatal error: 'gdk/gdkx.h' file not found
#include <gdk/gdkx.h>
         ^
1 error generated.
make[5]: *** [x11/libcmnpgtkhotkey_la-eggaccelerators.lo] Error 1
make[4]: *** [install-recursive] Error 1
make[3]: *** [install-recursive] Error 1
make[2]: *** [install-recursive] Error 1
make[1]: *** [install] Error 2
make: *** [install-recursive] Error 1

READ THIS: http://docs.brew.sh/Troubleshooting.html</details>

### CODE REVIEW

**Comment by yurikoles** [comment]
@Homebrew/maintainers - why there is no `<gdk/gdkx.h`

**Comment by yurikoles** [comment]
bump?

File: Formula/claws-mail.rb [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# Documentation: http://docs.brew.sh/Formula-Cookbook.html
+#                http://www.rubydoc.info/github/Homebrew/brew/master/Formula
+# PLEASE REMOVE ALL GENERATED COMMENTS BEFORE SUBMITTING YOUR PULL REQUEST!
```
Comment (MikeMcQuaid) [inline Formula/claws-mail.rb]: > PLEASE REMOVE ALL GENERATED COMMENTS BEFORE SUBMITTING YOUR PULL REQUEST!

File: Formula/claws-mail.rb [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+class ClawsMail < Formula
+  desc "Claws Mail e-mail client"
+  homepage "http://www.claws-mail.org"
+  url "http://www.claws-mail.org/download.php?file=releases/claws-mail-3.15.0.tar.xz"
+  sha256 "4e4d2d0d43f8ae3d4623408612f5979e9a697ccbc12038b80dd27802e868dc2e"
+
+  head 'http://git.claws-mail.org/readonly/claws.git'
+
+  depends_on :x11 # if your formula requires any X11/XQuartz components
```
Comment (MikeMcQuaid) [inline Formula/claws-mail.rb]: Can this be made to work at all without X11?

File: Formula/claws-mail.rb [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+#                http://www.rubydoc.info/github/Homebrew/brew/master/Formula
+# PLEASE REMOVE ALL GENERATED COMMENTS BEFORE SUBMITTING YOUR PULL REQUEST!
+
+class ClawsMail < Formula
+  desc "Claws Mail e-mail client"
+  homepage "http://www.claws-mail.org"
+  url "http://www.claws-mail.org/download.php?file=releases/claws-mail-3.15.0.tar.xz"
+  sha256 "4e4d2d0d43f8ae3d4623408612f5979e9a697ccbc12038b80dd27802e868dc2e"
+
+  head 'http://git.claws-mail.org/readonly/claws.git'
```
Comment (MikeMcQuaid) [inline Formula/claws-mail.rb]: Double quotes

File: Formula/claws-mail.rb [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+  def install
+    system "./configure", "--disable-debug",
+                          "--disable-dependency-tracking",
+                          "--disable-silent-rules",
+                          "--prefix=#{prefix}"
+    system "make", "install"
+  end
+
+  test do
+    system "true"
```
Comment (MikeMcQuaid) [inline Formula/claws-mail.rb]: Write a proper test.

**Review by MikeMcQuaid — CHANGES_REQUESTED** [review]

**Comment by MikeMcQuaid** [comment]
@yurikoles Don't know, sorry. Sounds like a missing dependency or missing configure option.

**Pull request closed without merging**